apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -45,11 +45,17 @@ class FormatType(Enum):
     CODE = "code"
 
 
+class BlockOrientationType(Enum):
+    DEFAULT = "default"
```

Comment:
I would prefer that the default actually be called "vertical" not just "default"

Reply:
disregard this if you end up just using `inline=true/false` you can just delete this enum altogether.